src/main/java/io/vertx/core/http/impl/Http2ServerRequest.java

```diff
@@ -70,6 +70,7 @@ public class Http2ServerRequest extends HttpServerRequestInternal implements Htt
   // Accessed on context thread
   private Charset paramsCharset = StandardCharsets.UTF_8;
   private MultiMap params;
+  private Boolean semicolonIsNormalCharInParams;
```

Comment:
we can use a primitive `boolean` here since `params != null` implies `semicolonIsNormalCharInParams` has been set

Reply:
Done.